src/redis.go

```diff
@@ -97,6 +88,22 @@ func main() {
 	fatalIfErr(i.Publish())
 }
 
+func metricSet(e *integration.Entity, eventType, hostname string, port int) *metric.Set {
+	strPort := strconv.Itoa(port)
+	if args.RemoteMonitoring {
```

Comment:
This `args` is from the global variables, should we add it as an argument to the function?